dataprofiler/tests/profilers/test_boolean_option.py

```diff
@@ -0,0 +1,82 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BooleanOption
+from dataprofiler.tests.profilers.test_base_option import TestBaseOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBooleanOption(TestBaseOption):
+    
+    @classmethod
+    def setUpClass(cls):
+        cls.option_class = BooleanOption    
+        
+    def test_init(self, *mocks):
+        option = self.get_options()
+        self.assertDictEqual({"is_enabled": True}, option.properties)
```

Comment:
we should also assert when setting `is_enabled=True` and `is_enabled=False`

Reply:
See lines 25-28. Those could be moved to test_init, but I want classes that inherit TestBooleanOption (like TestBaseColumnOptions) to check those (but not call super().test_init() since that would fail).
The purpose of test_init was to verify that default params (which may be assumptions in tests) haven't changed.

Reply:
I would consider these two separate tests as `_set_helper` should only test the `_set_helper` functionality and not whether the `__init__` properly constructs the option class. I just don't find it as intuitive that init tests would be contained elsewhere.

I think if you want to make the `init` more agnostic, you could do 1 of two things. 

1. only look at `options.is_enabled`
this validates class props are properly being set. We also make the test agnostic to the properties call which likely could be validated elsewhere (perhaps a separate test).
2. setup `test_init` in the abstract test class to take in the expected properties for evaluation

Reply:
of course other options exist as well, just thinking quickly here.

Reply:
That does make sense - it isn't intuitive to and init checks in the set_helper method. I modified the code slightly to better fit my intention with the tests. Now, it's still testing enabling and disabling the option with set/set_helper and verifying the init separately in the test_init test.